src/view/gui/AppearancePage.h

```diff
@@ -50,6 +58,22 @@ public slots:
     QFont m_SelectedFont, m_PreviousFont;
 	QString m_SelectedStyleSheet, m_PreviousStyleSheet;
 	ThemeType m_SelectedTheme, m_PreviousTheme;
+
+    /**
+     * @brief ApplySelectedAppearance applies whatever the currently selected font and stylesheet is to the global qApp.
+     */
+    void ApplySelectedAppearance();
+
+    /**
+     * @brief GetConfirmationFromUser raises a style-independent message box asking the user if they wish to keep their changes.
+     * @return  true if the user presses "Apply", false if "Cancel" or if 10 seconds pass
+     */
+    bool GetConfirmationFromUser();
+
+    /**
+     * @brief SetApplicationPreferences saves the current selections to the global ApplicationPreferences (allowing them to be saved to disk later).
+     */
+    void SetApplicationPreferences();
```

Comment:
name is a misnomer. We are not updating all the preferences..only the appearance.